setup.py

```diff
@@ -70,7 +70,9 @@ def run_tests(self):
         'lxml>=3.6.4',
         'asdf>=1.2.0',
         'scipy>=0.17',
-        'photutils>=0.4.0',
+        'photutils==0.4.0',
+        'pysiaf',
+        'pytest<=3.6'
```

Comment:
There is no need to include `pytest` unless you are writing a `pytest` plugin or something. This is because typical users won't care about the tests; they won't run the tests, so they don't need `pytest` to use your package. Adding it as a hard requirement here will also bring you pain.